src/models/sankeyChart.js

```diff
@@ -0,0 +1,239 @@
+nv.models.sankeyChart = function() {
+    "use strict";
+
+    // Sources:
+    // - https://bost.ocks.org/mike/sankey/
+    // - https://github.com/soxofaan/d3-plugin-captain-sankey
+
+    //============================================================
+    // Public Variables with Default Settings
+    //------------------------------------------------------------
+
+    var margin = {top: 5, right: 0, bottom: 5, left: 0}
+        , sankey = nv.models.sankey()
+        , width = 600
+        , height = 400
+        , nodeWidth = 36
+        , nodePadding =  40
+        , units =  'units'
+        ;
+    //============================================================
+
+
+    //============================================================
+    // Private Variables
+    //------------------------------------------------------------
+
+    var formatNumber = d3.format(',.0f');    // zero decimal places
+    var format = function(d) {
+        return formatNumber(d) + ' ' + units;
+    };
+    var color = d3.scale.category20();
+    var linkTitle = function(d){
+        return d.source.name + ' → ' + d.target.name + '\n' + format(d.value);
+    };
+    var nodeFillColor = function(d){
+        return d.color = color(d.name.replace(/ .*/, ''));
+    };
+    var nodeStrokeColor = function(d){
+        return d3.rgb(d.color).darker(2);
+    };
+    var nodeTitle = function(d){
+        return d.name + '\n' + format(d.value);
+    };
+
+    var showError = function(element, message) {
+        element.append('text')
+            .attr('x', 0)
+            .attr('y', 0)
+            .attr('class', 'nvd3-sankey-chart-error')
+            .attr('text-anchor', 'middle')
+            .text(message);
+    };
+
+
+    //============================================================
+
+    function chart(selection) {
+        selection.each(function(data) {
+
+            //============================================================
+            // Error handling
```

Comment:
I tried to remove them all, guess I missed some  :p 

originally there were like 20 of those per file with no real reasoning as to where they were, so it actually impeded readability.